pytorch_tabnet/tab_network.py

```diff
@@ -73,11 +73,11 @@ def __init__(self, input_dim, output_dim,
             Avoid log(0), this should be kept very low
         - mask_type: str
             Either "sparsemax" or "entmax" : this is the masking function to use
+        - prior_fn: callable
```

Comment:
Out of curiosity what kind of other priors would you like to use?

Reply:
Well, maybe I went a bit too far by making the ``prior_fn`` an argument here.
I just need it to be a class attribute so that we can change it on every epoch start. This is needed as at every epoch we need to mask different values of the inputs and the priors need to be adjusted accordingly as explained in section 4 of the paper.